src/NuGetGallery.Core/Cookies/CookieExpirationService.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Web;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace NuGetGallery.Cookies
+{
+    public class CookieExpirationService : ICookieExpirationService
+    {
+        private static readonly DateTime CookieExpirationTime = new DateTime(2010, 1, 1);
+
+        // Google Analytics cookies
+        private static readonly IReadOnlyList<string> GoogleAnalyticsCookies = new[]
+        {
+            "_ga",
+            "_gid",
+            "_gat",
+        };
+
+        // Application Insights cookies
+        private static readonly IReadOnlyList<string> ApplicationInsightsCookies = new[]
+        {
+            "ai_user",
+            "ai_session",
+        };
+
+        private readonly string Domain;
+        private readonly string RootDomain;
```

Comment:
Nit, please use `_abc` style for private fields:

```suggestion
        private readonly string _domain;
        private readonly string _rootDomain;
```